account_avatax/models/account_move.py

```diff
@@ -142,6 +142,9 @@ def _avatax_compute_tax(self, commit=False):
         if not avatax_config:
             # Skip Avatax computation if no configuration is found
             return
+        if not avatax_config.disable_tax_reporting:
+            # Skip Avatax Tax calculation if Tax reporting is disabled
+            return
```

Comment:
The `disable_tax_reporting`is supposed to not commit transaction, but should still perform the API calls.
So this is not functionally correct.
Am I seeing this right?

Reply:
Sorry, I meant to disable the check when disable_tax_calculation is disabled.

Reply:
Yes, removed this check as it is already checked in get_avatax_config_company